pkg/readers/github_org_reader.go

```diff
@@ -43,8 +44,13 @@ func (p *githubOrgReader) Name() string {
 	return "Github Organization Package Manifest Reader"
 }
 
+func (p *githubOrgReader) ApplicationName() (string, error) {
+	return githubOrgFromURL(p.config.OrganizationURL)
```

Comment:
This seems like misleading. Org should not be application name.Not sure what is a good default app name for a GitHub org

Reply:
hey, if we're ok with `vet-scanned-project` as default name, then maybe this can be `vet-scanned-safedep-projects` (safedep is org name)

Reply:
@OmkarPh Sounds good